Tabs/Tabs.jsx

```diff
@@ -21,10 +29,14 @@ class Tabs extends MaterialComponent {
   }
   componentDidMount() {
     this.MDComponent = new MDCTabBar(this.control);
+    setActiveTabIndex(defaultProps, this.props, this.MDComponent);
```

Comment:
This ignores the `activeTabIndex` prop on initial mount and only uses the prop on update